src/main/java/io/github/jazorp/Validators.java

```diff
@@ -33,6 +37,25 @@
     // =====================
     // Validators
 
+	public static ValidationThunk memberOf(String field, final Object value, Iterable<Object> values) {
+		Collection<Object> collection = null;
+		if (values instanceof Collection) {
```

Comment:
Instead of casting and converting the values, let's use the `Iterator` obtained from the `Iterable` to check the presence of the value. 

This library can be considered as a fairly low level one, so any unnecessary overhead should be avoided.